src/server/table_hotspot_policy.h

```diff
@@ -98,23 +44,44 @@ class hotspot_calculator
     hotspot_calculator(const std::string &app_name,
                        const int partition_num,
                        std::unique_ptr<hotspot_policy> policy)
-        : _app_name(app_name), _points(partition_num), _policy(std::move(policy))
+        : _app_name(app_name),
+          _points(partition_num),
+          _policy(std::move(policy)),
+          kHotPartitionT(0),
+          kHotRpcT(0)
     {
+        ddebug("hotspot_calculator construct!");
         init_perf_counter(partition_num);
```

Comment:
Move to the last line of the construct function and add some necessary config message